guide/english/html/attributes/input/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Input
 ---
-## Input
-The HTML `<input>` tag is used within a form to declare an input element.
-It allows the user to enter data.
+
+# Input
+The HTML `<input>` tag is used to create an element which allows a user to input data. Typically this would be within a `<form>` element, although it's not required.
 
 ## Example
```

Comment:
I know this is not your change, but while you are at it, I hope you don't mind changing this to `###`.